internal/backends/qt/qt_window.rs

```diff
@@ -1375,7 +1375,19 @@ impl QtItemRenderer<'_> {
     }
 }
 
-cpp_class!(pub(crate) unsafe struct QWidgetPtr as "std::unique_ptr<QWidget>");
+cpp! {{
+    struct QWidgetDeleteLater
+    {
+        void operator()(QWidget *widget_ptr)
+        {
+            widget_ptr->hide();
+            widget_ptr->disconnect();
```

Comment:
Why disconnect? I'm afraid this could cause leaks or bug if something (eg, th QStyle) use that to connect to the destroyed signal or something.
Unless you have a good reason to have that, I'd remove that line

Reply:
I hadn't thought of that aspect. My original intention was to have the object be deleted quickly so it doesn't hang around in memory, but I'm assuming that's preferable to a permanent leak or crash down the line.